README.md

```diff
@@ -15,12 +15,16 @@ applications.  ISA-L includes:
   implementations.
 * Compression - Fast deflate-compatible data compression.
 * De-compression - Fast inflate-compatible data compression.
+* igzip - A command line application like gzip, accelerated with ISA-L. (Not 
+  available on Windows. Windows users can use the `python -m isal.igzip` 
```

Comment:
The igzip tool does work on windows.  You need the getopt and utime libs currently to build igzip.exe but it is possible.  Info on the python compression tool is probably better put in doc/ or wiki.  I would suggest that we start a doc/igzip_cli.md with info on both and put a link to it in "Also see:" section after wiki.

Reply:
In that case I don't think it is necessary to mention the python-isal provided alternative.
I had some troubles getting igzip.exe to compile since I couldn't find `utime.h` anywhere. But now I read somewhere that it should be `SYS\utime.h` on windows apparently. Getopt is part of vcpks so that should not be a problem.

I will try my best to compile it, and once I succeed will list the steps in the windows compile instructions, as well as providing igzip.exe in the conda package (currently only the dll is in the conda package).

Reply:
Managed to get  further by installing the conda getopt-win32 package and by patching igzip_cli.c to include `sys\utime.h` instead of `utime.h`. 
But then the following error occurs:
```
programs\igzip_cli.c(38): fatal error C1083: Cannot open include file: 'unistd.h': No such file or directory
```
`unistd.h` seems to be very POSIX-specific. I have not found any information on how to get this on Windows.

@gbtucker Is there a working windows igzip.exe compilation script that I can use as an example somewhere?

Reply:
That's right.  We tried to follow the functionality exactly for the standard gzip tool and there are some particulars requiring isatty() and thus unistd.h.  Getting an .exe is simple enough with mingw cross tools (or perhaps cygwin) and this is what I was remembering.  I don't know all the dependencies needed when building with MS tools.

Reply:
Thanks. It seems that building igzip.exe is quite a project. I removed the python -m isal.igzip suggestion. I will try to build igzip.exe and publish the full steps here in a separate PR once I succeed. It seems that crossbuilding is the way to go, as mingw does not find unistd.h in a windows environment. If I succeed I will also include igzip.exe in the binary package on conda-forge so I can point windows users there.